lib/Partitioner/Partitioner.cpp

```diff
@@ -935,6 +945,57 @@ llvm::Error Partitioner::createDAGWithoutPartition(
   return llvm::Error::success();
 }
 
+llvm::Error
+Partitioner::QuantizationProfilingPartition(CompilationContext &cctx) {
```

Comment:
This function seems to be very similar to other functions in the partitioner. Is it possible to have some code-reuse?